src/Equivalence_Reasoner.thy

```diff
@@ -126,10 +126,10 @@ text \<open>
   and solve the resulting goals using the Simplifier. The translation from equivalences to
   equalities that are ready for simplification works as follows:
 
-    \<^item> First, each equivalence \<^term>\<open>R x y\<close> is turned into \<^term>\<open>[x]\<^bsub>R\<^esub> = [y]\<^bsub>R\<^esub>\<close>.
+    \<^item> First, each equivalence \<^term>\<open>R t\<^sub>1 t\<^sub>2\<close> is turned into \<^term>\<open>[t\<^sub>1]\<^bsub>R\<^esub> = [t\<^sub>2]\<^bsub>R\<^esub>\<close>.
```

Comment:
For the sake of consistency, shouldn't `t⇩1` and `t⇩2` be changed to `t` and `t'`, respectively?

Reply:
I intended to always use `t⇩1` and `t⇩2`, but found it awkward to talk about “each equivalence class construction `[t⇩1]⇘R⇙`”, with `t⇩2` only being introduced slightly later. Therefore, I chose `t` and `t'` for rewrite rules and compatibility rules (which are internally used almost like rewrite rules), and reserved `t⇩1` and `t⇩2` for the situation where I talk about an equivalence to prove.

Perhaps the above-mentioned phrase wouldn’t be that awkward. Therefore, I’ll just change `t` and `t'` to `t⇩1` and `t⇩2`, following the approach I had originally intended to follow.